apps/web/src/blocks/FeatureSection/index.tsx

```diff
@@ -0,0 +1,41 @@
+// Types
+import type { FeatureSection as PayloadType } from '@mono/types/payload-types';
+
+// Libraries
+import React from 'react';
+import dynamic from 'next/dynamic';
+
+// Components
+import Wrapper from '@mono/web/components/Wrapper';
+
+export type FeatureSectionType = Omit<PayloadType, 'blockType'>;
+
+const Variant = (variant: string) => {
+  // Can extend this to accept different props based on the variant
+  // const Variant = (variant: string, payload: any) => {
+  const defaultOpts = {
+    suspense: true,
+    ssr: true
+  };
+
+  // Add logic here if you want to load different components based on the variant
+  // let options = {
+  //   ...defaultOpts
+  // };
+
+  return dynamic(() => import(`./variations/Variant${variant}`), {
```

Comment:
@tubstrr i think it's frowned upon to use dynamic variables in import paths. it's throwing compiler warnings in the console.

from the next docs:

> Good to know: In import('path/to/component'), the path must be explicitly written. It can't be a template string nor a variable. Furthermore the import() has to be inside the dynamic() call for Next.js to be able to match webpack bundles / module ids to the specific dynamic() call and preload them before rendering. dynamic() can't be used inside of React rendering as it needs to be marked in the top level of the module for preloading to work, similar to React.lazy.